roles/openshift_logging_elasticsearch/files/crd.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: elasticsearchs.elasticsearch.redhat.com
```

Comment:
can you please update this? I've rebased to the latest master of operator-sdk, where they changed plurals to `elasticsearches`
https://github.com/ViaQ/elasticsearch-operator/blob/master/deploy/crd.yaml#L4